src/router/index.js

```diff
@@ -49,19 +50,19 @@ const routes = [
         component: () => import("../profile/views/Profile.vue"),
       },
       {
-        path: "products",
+        path: ":companyId/products",
         component: () => import("../views/Products.vue"),
       },
       {
-        path: "sales",
+        path: ":companyId/sales",
         component: () => import("../views/Sales.vue"),
       },
       {
-        path: "fees",
+        path: ":companyId/fees",
         component: () => import("../views/Fees.vue"),
       },
       {
-        path: "employees",
+        path: ":companyId/employees",
         component: () => import("../views/Employees.vue"),
```

Comment:
```suggestion
        component: () => import("../employees/views/EmployeesDashboard.vue"),
```

we can rename component `EmployeesDashboard` to `Employees`